pkg/api/frontendsettings.go

```diff
@@ -181,7 +181,7 @@ func (hs *HTTPServer) getFrontendSettingsMap(c *models.ReqContext) (map[string]i
 		"alertingNoDataOrNullValues": setting.AlertingNoDataOrNullValues,
 		"alertingMinInterval":        setting.AlertingMinInterval,
 		"autoAssignOrg":              setting.AutoAssignOrg,
-		"verfiyEmailEnabled":         setting.VerifyEmailEnabled,
+		"verifyEmailEnabled":         setting.VerifyEmailEnabled,
```

Comment:
This could be an API Change, in which case it's probably best to drop or add a shim.